### ISSUE TITLE
Does not trigger react's valuelink function on change

### ISSUE THREAD

**Issue by parshwa** [eternicode/bootstrap-datepicker#1832]
I am using bootstrap-datepicker with react. Here is the code which is rendered:
`<input type="text" valueLink={this.dateValueLink('date-0')} className="form-control tripDatePicker" placeholder="Select Date"/>`

I am initiating datepicker in componentDidMount
`componentDidMount: function () { $('.tripDatePicker').datepicker({ startDate: "today", autoclose: true, format: "dd/mm/yyyy" }); }`

Datepicker comes properly and when I select a date, it shows up in input field. But my valuelink's requestChange function does not fire.
I know there is option of using event like `.on('changeDate')`, but it will make things very difficult for me.

Is there anyway by which I can use valuelink with this datepicker?

**Comment by aocneanu** [comment]
It would be nice if along with "changeDate" event the DOM's "change" event would be triggered too. Using bootstrap-datepicker with vue.js is problematic too.

**Comment by Azaret** [comment]
Yeah the datepicker is build up on jQuery, so it use jQuery event propagation.
I'll tag this as a maybe future feature, as building the plugin in vanilla JS and add jQuery API is more popular these days.
Add this snippet for future references.

```javascript
var input = $('input'), result = $('#A');

input.datepicker();
input.on('change', function () {
    result.append('<div style="color:red;">jQuery event trigerred</div>');
});
input.get(0).onchange = function () {
    result.append('<div style="color:red;">JS function trigerred</div>');
}
addEventHandler(input.get(0), 'change', function() {
    // not trigerred
    result.append('<div style="color:red;">JS event trigerred</div>');
});
```
